src/pages/AddLiquidity/index.tsx

```diff
@@ -511,10 +523,12 @@ export default function AddLiquidity({
             </RowBetween>
           )}
         {mustCreateSeparately && (
-          <ButtonError onClick={onCreate}>
-            <Text fontWeight={500}>
-              <Trans>Create</Trans>
-            </Text>
+          <ButtonError onClick={onCreate} disabled={!isValid || attemptingTxn || !position}>
+            {attemptingTxn ? (
+              <Dots>Confirm Create</Dots>
+            ) : (
+              <Text fontWeight={500}>{errorMessage ? <Trans>{errorMessage}</Trans> : <Trans>Create</Trans>}</Text>
```

Comment:
trans doesn't work with just a single substitution, i.e. <Trans>{errorMessage}</Trans> will not be translated. the error message itself has to be translated

Reply:
Oh i see now

Reply:
Actually where would we do the translation here then ?

Reply:
`errorMessage = <Trans>something went wrong</Trans>`

Reply:
`errorMessage` is already translated wrapped in a `t` macro inside the hook so nothing to do.